modules/reference/pages/rpk/rpk-topic/rpk-topic-consume.adoc

```diff
@@ -7,296 +7,203 @@ according to `--format`, and prints them to `STDOUT`. The output formatter
+formatting option described above. No compression is `0`, gzip is `1`, etc.
 
-[.no-copy]
+[,bash]
 ----
-     %a{timestamp-type}
-     %a{timestamp-type;big64}
+%a{timestamp-type}
+%a{timestamp-type;big64}
 ----
 
-The record's timestamp type is printed as -1 for very old records (before
-timestamps existed), 0 for client generated timestamps, and 1 for broker
-generated timestamps. Number formatting can be controlled with ";number".
+The record's timestamp type is printed as:
+*  `-1` for very old records (before timestamps existed)
+*  `0` for client generated timestamps
+*  `1` for broker generated timestamps.
+
+NOTE: Number formatting can be controlled with `;number`.
 
-[.no-copy]
+[,bash]
 ----
-     %a{transactional-bit}
-     %a{transactional-bit;bool}
+%a{transactional-bit}
+%a{transactional-bit;bool}
 ----
 
-Prints 1 if the record a part of a transaction or 0 if it is not.
-Number formatting can be controlled with `;number`.
+Prints `1` if the record is a part of a transaction, or `0` if it is not.
 
-[.no-copy]
+[,bash]
 ----
-     %a{control-bit}
-     %a{control-bit;bool}
+%a{control-bit}
+%a{control-bit;bool}
 ----
 
-Prints 1 if the record is a commit marker or 0 if it is not.
-Number formatting can be controlled with ";number".
+Prints `1` if the record is a commit marker or `0` if it is not.
 
 === Text
 
 Text fields without modifiers default to writing the raw bytes. Alternatively,
 there are the following modifiers:
 
-[.no-copy]
-----
-    %t{hex}
-    %k{base64}
-    %v{base64raw}
-    %v{unpack[<bBhH>iIqQc.$]}
-----
+[cols=",",]
+|===
+|Modifier |Description
 
-The hex modifier hex encodes the text, the base64 modifier base64 encodes the
-text with standard encoding, and the base64raw modifier encodes the text with
-raw standard encoding. The unpack modifier has a further internal
-specification, similar to timestamps above:
+|`%t\{hex}` |Hex encoding
 
-[.no-copy]
-----
-    x    pad character (does not parse input)
-    <    switch what follows to little endian
-    >    switch what follows to big endian
-
-    b    signed byte
-    B    unsigned byte
-    h    int16  ("half word")
-    H    uint16 ("half word")
-    i    int32
-    I    uint32
-    q    int64  ("quad word")
-    Q    uint64 ("quad word")
-
-    c    any character
-    .    alias for c
-    s    consume the rest of the input as a string
-    $    match the end of the line (append error string if anything remains)
-----
+|`%k\{base64}` |Base64 encoding
+
+|`%k\{base64raw}` |Base64 encoding raw
+
+|`%v{unpack[<bBhH>iIqQc.$]}` |The unpack modifier has a further internal
+specification, similar to timestamps above.
+|===
+
+The hex modifier hex encodes the text, the `base64` modifier base64 encodes the
+text with standard encoding, and the `base64raw` modifier encodes the text with
+raw standard encoding. The unpack modifier has a further internal
+specification, similar to timestamps above: |Match the end of the line (append error string if anything
```

Comment:
@Deflaimun is this (_|Match the end..._) a typo?

Reply:
I'll remove this paragraph since we already have that info on the table